src/main/webapp/modal-overlay.css

```diff
@@ -151,6 +151,33 @@ a:hover {
   color: rgba(230, 230, 230); /* Grey. */ 
 }
 
+.button-group button {
+  cursor: pointer;
+  padding: 10px 24px;
```

Comment:
CSS should be alphabetical!

Reply:
Shoot! I made the comment on a previous PR. I should have been more careful.